lane.py

```diff
@@ -603,7 +602,6 @@ class DatabaseBackedFacets(Facets):
         FacetConstants.ORDER_TITLE : Edition.sort_title,
         FacetConstants.ORDER_AUTHOR : Edition.sort_author,
         FacetConstants.ORDER_LAST_UPDATE : Work.last_update_time,
-        FacetConstants.ORDER_RANDOM : Work.random,
```

Comment:
This is the bit that stops random order from working for DatabaseBackedLists. We can't sort randomly because there's no longer a database field to sort on.